### PR TITLE
Feature/sidhant/step5

### LINKED ISSUE #14: Step 5 Component
- Remeber to display the payment amount according to the Path

### PR DESCRIPTION
**Pull request by sidhantrohatgi**
## Tracking Info

Resolves #14 

## Changes

Added Step 5

- TODO

## Testing

Ran the frontend and checked if the changes were reflected according to the figma

## Confirmation of Change

Screenshot 2024-05-09 at 18 03 34

### CODE REVIEW

File: frontend/src/components/Step5.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  };
+
+  return (
+    <>
+      <form id="step5-form" onSubmit={onSubmit}>
+        <div className={styles.formContainer}>
+          <hr />
+          <div className={styles.formSection}>
+            <p>
+              To pay via <strong className={styles.red}> credit card </strong>: Navigate to the
```
Comment (Miyuki-L) [inline frontend/src/components/Step5.tsx:28]: There is a extra space btw card and :

**Review by Miyuki-L — COMMENTED** [review]
Looking good. Just a few things to clean up and also the button in the bottom navigator when on payment page

File: frontend/src/components/Step5.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    const form = event.target as HTMLFormElement;
+    if (!form.checkValidity()) {
+      return;
+    }
+    next();
+  };
+
+  return (
+    <>
+      <form id="step5-form" onSubmit={onSubmit}>
```
Comment (Miyuki-L) [inline frontend/src/components/Step5.tsx:23]: Missing the border/shaow between the element and the background. 
image


See how there is a shadow/border on the top component but there isn't one on the bottom one

File: frontend/src/components/Step5.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import { Button } from "./Button.tsx";
+import styles from "./Step5.module.css";
+
+// Pass in Application's next function
+export type StepProps = {
+  next: () => void;
+};
+
+export const Step5: React.FC<StepProps> = ({ next }: StepProps) => {
+  // Checks if all Required Inputs are filled before moving on to next step
```
Comment (leo3friedman) [inline frontend/src/components/Step5.tsx:10]: I'm a little confused as to why Step5 is a form. Also, why we are checking that all required inputs are filled before moving on? From what I can tell the step has no inputs. Thanks

**Review by leo3friedman — CHANGES_REQUESTED** [review]
I had one question about why we use a form for Step5. Also, looks like you may want to merge main into this branch + resolve conflicts. Besides that, looks good!

**Comment by Miyuki-L** [comment]
#42

**Pull request closed without merging**